unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB_GameObject/Tests/APK_AB_GameObject_Promise_Should.cs

```diff
@@ -142,8 +142,7 @@ public IEnumerator FailCorrectlyWhenLoadingABWithoutGameObjects()
 
             yield return prom;
 
-            Assert.IsTrue(failed == true, "The broken paladin AssetBundle contains no GameObjects and should fail!");
-            LogAssert.Expect(LogType.Exception, new Regex("^.*?AB sub-promise asset or container is null"));
+            Assert.IsTrue(failed, "The broken paladin AssetBundle contains no GameObjects and should fail!");
```

Comment:
Wtf is a broken paladin, can we improve this error log?

Reply:
The person that did the tests in the past, decided to use this "Broken Paladin" as a test that should fail because it does not exist.
We can change it but, as is a test for "FailCorrectly", has no interference in the project in any case.